pymc3/tests/test_step.py

```diff
@@ -238,3 +240,36 @@ def test_binomial(self):
             Binomial('x', 10, 0.5)
             steps = assign_step_methods(model, [])
         self.assertIsInstance(steps, Metropolis)
+
+
+class TestSampleEstimates(SeededTest):
+    def test_parameter_estimate(self):
+        alpha_true, sigma_true = 1, 0.5
+        beta_true = np.array([1, 2.5])
+
+        size = 100
+
+        X1 = np.random.randn(size)
+        X2 = np.random.randn(size) * 0.2
+        Y = alpha_true + beta_true[0] * X1 + beta_true[1] * X2 + np.random.randn(size) * sigma_true
+
+        with Model() as model:
+            alpha = Normal('alpha', mu=0, sd=10)
+            beta = Normal('beta', mu=0, sd=10, shape=2)
+            sigma = Uniform('sigma', lower=0.0, upper=1.0)
+            mu = alpha + beta[0] * X1 + beta[1] * X2
+            Y_obs = Normal('Y_obs', mu=mu, sd=sigma, observed=Y)
+
+            for step_method in (NUTS(), Metropolis(),
+                                [Slice([alpha, sigma]), Metropolis([beta])]):
+                trace = sample(1000, step=step_method, progressbar=False)
+
+                np.testing.assert_array_almost_equal(np.median(trace.beta, 0), beta_true, decimal=1)
+                np.testing.assert_array_almost_equal(np.median(trace.alpha), alpha_true, decimal=1)
+                np.testing.assert_array_almost_equal(np.median(trace.sigma), sigma_true, decimal=1)
+                # Using the Normal test from SciPy `scipy.stats.normaltest`
+                # See `https://github.com/scipy/scipy/blob/v0.18.1/scipy/stats/stats.py#L1352`
+                _, test_normal_beta = stats.normaltest(trace.beta[:,1])
+                _, test_uniform = stats.normaltest(trace.sigma)
+                np.testing.assert_array_less(test_normal_beta, 0.005)
```

Comment:
But I thought we realized that these should *not* be significant, so this means the posteriors are *not* normal, indicating a problem with our sampling. CC @fonnesbeck @jsalvatier